src/redux/books/fetchApi.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/8uAGkqGHBMzF9G5Da8NZ/books';
+const fetchData = createAsyncThunk('Booklists/get', async (arg, { rejectWithValue }) => {
```

Comment:
- [ ] I ❤️ like this, awesome use of reusable components.

Reply:
@joskalenda  Thanks bro ❤️